src/GraphQL/Internal/Validation.hs

```diff
@@ -58,7 +58,7 @@ module GraphQL.Internal.Validation
   , findDuplicates
   ) where
 
-import Protolude hiding ((<>))
+import Protolude hiding ((<>), throwE)
```

Comment:
Do we have a `throwE` collision as well?

Reply:
it looks like it. After hiding the `Type` from `protolude`, I got the same error with this `throwE`